README.md

```diff
@@ -1,53 +1,72 @@
 # maven-settings-action
+
 [![Test](https://github.com/s4u/maven-settings-action/workflows/Test/badge.svg)](https://github.com/s4u/maven-settings-action/actions?query=workflow%3ATest)
 [![Audit](https://github.com/s4u/maven-settings-action/workflows/Audit/badge.svg)](https://github.com/s4u/maven-settings-action/actions?query=workflow%3AAudit)
 
 
+
 This action setup maven environment for use in action by:
  - create maven settings.xml
  - set ```interactiveMode``` to false - useful in CI system
  - after job finish generated settings.xml will be removed to prevent cache or left sensitive data on build system
  - add server to servers with id=github, username=$GITHUB_ACTOR and password=$GITHUB_TOKEN
 
-# Contributions
+## Contributions
+
 - Contributions are welcome!
 - Give :star: - if you want to encourage me to work on a project
 - Don't hesitate create issue for new feature you dream of or if you suspect some bug
 
-# Project versioning
+## Project versioning
 Project use [Semantic Versioning](https://semver.org/).
 We recommended to use the latest and specific release version.
 
 In order to keep your project dependencies up to date you can watch this repository *(Releases only)*
 or use automatic tools like [Dependabot](https://dependabot.com/).
 
 
- # Usage
+## Usage
+
 See [action.yml](action.yml)
 
 Create default ```settings.xml```:
+
 ```yml
 steps:
 - uses: s4u/maven-settings-action@v2
 ```
 
 Create ```settings.xml``` with server section:
+
 ```yml
 steps:
 - uses: s4u/maven-settings-action@v2
   with:
     servers: '[{"id": "serverId", "username": "username", "password": "password"}]'
 ```
 
+Create ```settings.xml``` with [Jfrog Artifactory](https://www.jfrog.com/confluence/display/JFROG/Maven+Repository) in repository list
+
+```yml
+steps:
+- uses: s4u/maven-settings-action@v2.0
+  with:
+    servers: '[{"id": "central", "username": "username", "password": "password"}]'
+    servers: '[{"id": "snapshots", "username": "username", "password": "password"}]'
```

Comment:
I guess credentials for both artifactory repository id `central` and `snapshot` will be the same.
So maybe better add params: `artifactory_username`, `artifactory_password` and in code create two servers item for theam.